airflow/contrib/operators/dataproc_operator.py

```diff
@@ -256,7 +258,7 @@ def _get_init_action_timeout(self):
             "DataprocClusterCreateOperator init_action_timeout"
             " should be expressed in minutes or seconds. i.e. 10m, 30s")
 
-    def _build_cluster_data(self):
+    def _build_cluster_data(self):  # pylint: disable=too-many-branches
```

Comment:
I'd split this method  rather than disable the error. It should be fairly easy with "extract method" refactor from IntelliJ